app/experimenter/experiments/tests/test_views.py

```diff
@@ -256,6 +256,30 @@ def test_list_filters_by_search_text(self):
             set(second_response_context["experiments"]), set([exp_3])
         )
 
+    def test_filters_by_review_in_qa(self):
+        ExperimentFactory.create_with_variants(
+            review_qa_requested=True, review_qa=False
+        )
+        ExperimentFactory.create_with_variants(
+            review_qa_requested=False, review_qa=False
+        )
+        ExperimentFactory.create_with_variants(
+            review_qa_requested=True, review_qa=None
+        )
+
+        filter = ExperimentFilterset(
+            {"reviews": "in_qa"}, queryset=Experiment.objects.all()
+        )
+
+        self.assertEqual(
+            set(filter.qs),
+            set(
+                Experiment.objects.filter(
+                    review_qa_requested=True, review_qa=False
```

Comment:
I would just assign each of the test experiments to variables and then refer to them here, ie

    exp1 = ExperimentFactory.create(......
    ...
    self.assertEqual(set(filter.qs), set([expN, ....])

that way it's more explicit exactly which experiments are matching/not matching without needing to query the db again.

Reply:
gotcha!